groups/bsl/bslmt/bslmt_testutil.t.cpp

```diff
@@ -1530,239 +1496,239 @@ int main(int argc, char *argv[])
                       "====================================================\n";
 
         enum {
-            BUFFER_SIZE    = 1024,  // size of the buffer used to store
-                                    // captured output
+	    BUFFER_SIZE  = 1024,  // size of the buffer used to store
```

Comment:
Yeah, if `k_REPEAT` has `k_`, I think this should have `k_`.  Or neither.

Reply:
Yikes! This file has many more cleanups than I remember, but the NUM_ITERATIONS -> k_REPEAT was the only essential renaming, as we had to reduce name length so that 2 or 3 assertions would fit on a single line.

As long as adding the 'k_' does not introduce new line wrapping problems (that affect correctness) then switching to 'k_BUFFER_SIZE' for consistency would make sense - agreed.

Reply:
That is the one I can see.  I do not know how many more constants are in this file.  Maybe it is easier to remove the `k_` from the changed one.  Unless someone makes a `REPEAT` macro in a system header...

Reply:
Looking at the full file the inconsistency is already present, so I withdraw my comment, we did not *introduce* it.